### ISSUE TITLE
IWSLT preparation script is overwriting prepared validation data

### ISSUE THREAD

**Issue by bricksdont** [pytorch/fairseq#1641]
## 🐛 Bug

The preparation script for IWSLT 2017 multilingual data wants to preprocess several validation sets, but in fact, only the last validation set is saved for each language pair.

Specific lines of code in question: https://github.com/pytorch/fairseq/blob/master/examples/translation/prepare-iwslt17-multilingual.sh#L85-98

Should all validation sets be removed except one, or would you like to keep them all?
